testapi.pm

```diff
@@ -681,6 +681,22 @@ sub console($) {
     }
 }
 
+sub assert_shutdown(;$) {
+    my ($timeout) = @_;
+    $timeout //= 60;
+    bmwqemu::log_call('assert_shutdown', timeout => $timeout);
+    while ($timeout >= 0) {
+        my $status = $bmwqemu::backend->status() // '';
+        if ($status eq 'shutdown') {
+            $autotest::current_test->take_screenshot('ok');
+            return;
+        }
+        --$timeout;
+        sleep 1 if $timeout >= 0;
+    }
+    $autotest::current_test->take_screenshot('fail');
+    croak "Machine didn't shut down!";
+}
```

Comment:
why croak? we already log the call and a die will do. I don't really like using die in some calls and croak in others.